python/feldera/pipeline.py

```diff
@@ -285,31 +213,17 @@ def start(self):
         """
         .. _start:
 
-        Start the pipeline.
+        Starts this pipeline.
 
         :raises RuntimeError: If the pipeline returns unknown metrics.
         """
 
-        pipeline = Pipeline(
-            self.pipeline_name,
-            sql=self.ddl,
-            description=self.pipeline_description,
-            program_config={
-                'profile': self.compilation_profile.value,
-            },
-            runtime_config=self.runtime_config.__dict__,
-        )
-
-        self.client.create_pipeline(pipeline)
-
-        current_state = self.pipeline_status()
-        if current_state not in [PipelineStatus.NOT_FOUND, PipelineStatus.SHUTDOWN]:
-            raise RuntimeError(f"pipeline in state: {str(current_state.name)} cannot be started")
+        status = self.status()
+        if status not in [PipelineStatus.NOT_FOUND, PipelineStatus.SHUTDOWN]:
```

Comment:
This is just to make it nicer to work with within the python SDK, instead of checking for an error, it is more convenient when it is an enum variant.

Reply:
The errors will occur though in any case? E.g., when a pipeline which does not exist is shutdown, the response will still be an HTTP error NOT FOUND.

Reply:
Yes. 

The variant is just to handle the 404 and propagate it nicely up the stack instead of using a try block around the caller of `Pipeline.status`.

Reply:
But when a `Pipeline` object is instantiated, it should never be NOT FOUND unless there were concurrent things happening or the Python script issued `delete()` by itself and keeps using it. In these cases, it should just return an error

Reply:
Oh I get what you mean. Yeah, I copied this over from the previous implementation without much thought. 
In our current implementation, yeah it cannot be NOT FOUND.